### PR TITLE
Bump CUB and update build scripts for testing changes.

### PR DESCRIPTION
**Pull request by allisonvacanti**
Updates and tests NVIDIA/cub#403.

### CODE REVIEW

**Comment by allisonvacanti** [comment]
run tests

**Comment by allisonvacanti** [comment]
run tests

**Comment by allisonvacanti** [comment]
run tests

**Comment by allisonvacanti** [comment]
run tests

**Comment by allisonvacanti** [comment]
run tests

**Comment by allisonvacanti** [comment]
run tests

**Comment by allisonvacanti** [comment]
run tests

**Comment by allisonvacanti** [comment]
run tests

**Comment by allisonvacanti** [comment]
DVS CL: 30772865

File: ci/common/build.bash [hunk 1]
```diff
@@ -188,9 +188,6 @@ case "${COVERAGE_PLAN}" in
     append CMAKE_FLAGS "-DTHRUST_MULTICONFIG_ENABLE_SYSTEM_CUDA=ON"
     append CMAKE_FLAGS "-DTHRUST_MULTICONFIG_WORKLOAD=SMALL"
     append CMAKE_FLAGS "-DTHRUST_INCLUDE_CUB_CMAKE=ON"
-    append CMAKE_FLAGS "-DCUB_ENABLE_THOROUGH_TESTING=OFF"
```
Comment (senior-zero) [inline ci/common/build.bash:191]: Should these flags be deleted from `ci/common/build.bash` as well?

**Review by senior-zero — APPROVED** [review]

File: ci/common/build.bash (hunk 1 above)
Comment (allisonvacanti) [reply ci/common/build.bash:191]: I'm not sure what you mean -- this patch does remove the CUB testing flags from that file.

**Comment by allisonvacanti** [comment]
DVS CL: 301781720

run tests

**Pull request merged**